docs/book/orders.rst

```diff
@@ -4,117 +4,103 @@
 Orders
 ======
 
-*Order* model is one of the most important in Sylius, it represents the order placed via your store! It has a very consistent and clear API, which allows you to easily manipulate and process orders.
+**Order** model is one of the most important in Sylius, where many concepts of e-commerce meet.
+It represents an order that can be either placed or in progress.
 
-Customer Reference
-------------------
+**Order** holds a collection of **OrderItem** instances, which represent products from the shop,
+as it physical copies, with chosen variants and quantities.
 
-*Order* holds a reference to specific *User*, which is available through ``getUser()`` method:
+The Order is **assigned to the channel** in which it has been created. Moreover the **language** the customer was using
+and the **currency with its exchange rate** at the moment of creation are saved.
+
+How to create an Order programmatically?
+----------------------------------------
+
+To programmatically create an Order you will of course need a factory.
 
 .. code-block:: php
 
-    echo $order->getUser()->getEmail(); // john@example.com
+    /** @var FactoryInterface $order */
+    $orderFactory = $this->get('sylius.factory.order');
+
+    /** @var OrderInterface $order */
+    $order = $orderFactory->createNew();
 
-When creating order programatically, you can define the user yourself:
+Then get a channel to which you would like to add your Order. You can get it from the context or from the repository by code for example.
 
 .. code-block:: php
 
-    $order = $this->get('sylius.repository.order')->createNew();
-    $john = $this->get('sylius.repository.user')->find(3);
+    /** @var ChannelInterface $channel */
+    $channel = $this->container->get('sylius.context.channel')->getChannel();
 
-    $order->setUser($john);
+    $order->setChannel($channel);
 
-*Order* may not have reference to *User* in case when *Order* was created by guest.
+What is more the proper Order instance should also have the **Customer** assigned.
```

Comment:
Here ae are describing teh concept of Order. Not Cart. That is why we need a Customer.